CMakeLists.txt

```diff
@@ -69,14 +69,16 @@ endif()
 
 include(PrecompiledHeader)    #Precompiled Header helper macros
 
-# libCurl isn't smart enough to detect this for us, so we have to configure it ourselves
-option(CURL_IS_STATIC "Using the static version of libcurl?" ON)
-if(CURL_IS_STATIC)
-    add_definitions(-DCURL_STATICLIB)
-endif(CURL_IS_STATIC)
-
-# Same story for expat
-option(EXPAT_IS_STATIC "Using the static version of libexpat" ON)
+if(WIN32)
```

Comment:
If you read the commit comment, the CURL_IS_STATIC and EXPAT_IS_STATIC flags are meaningless outside of WIN32 (i.e. they only control whether to use `__declspec(dllimport)` / `__declspec(dllexport)`).  I discovered that I've been leaving them set to the default (ON) for ages even though my system libs are not static.